assets/javascript/app.js

```diff
@@ -1,4 +1,16 @@
 $(document).ready(function(){
-	var questions = [questionOne, questionTwo, questionThree, questionFour, questionFive]
-})
+
+	var correctAnswers = ["The Soup Nazi", "Brett Leonard"];
+
+
+	$('#submit').on('click', function() {
+
+
+		console.log('question 1 answer =', $('#question_1 input:radio:checked').attr('value'));
+		console.log('is question 1 correct? ', ($('#question_1 input:radio:checked').attr('value') == correctAnswers[0]));
+
```

Comment:
This is just an example of pulling some data from the dom when you click on submit. Searching for the dom element #question_1 then searching for an input radio that is currently checked inside of that div gives you whatever the user picked for question_1.